Tests/LoggingTests/LoggingTest.swift

```diff
@@ -125,6 +125,52 @@ class LoggingTest: XCTestCase {
                                                    "nested-list": ["l1str", ["l2str1", "l2str2"]]])
     }
 
+    // Example of custom "box" which may be used to implement "render at most once" semantics
+    // Not synchronized, thus should not be shared across threads
```

Comment:
nit: I think `synchronized` is not really used outside of the `JVM` world. Maybe `Not thread-safe` would be better.

Reply:
Ah, good catch :)

Reply:
Addressed, thanks!